superbench/benchmarks/micro_benchmarks/cudnn_test/cudnn_function_helper.h

```diff
@@ -0,0 +1,201 @@
+ */
+
+#pragma once
+
+#include <fstream>
+#include <iostream>
+#include <limits>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include <nlohmann/json.hpp>
+
+#include "cudnn_benchmark.h"
+#include "cudnn_function.h"
+
+using json = nlohmann::json;
+
+/**
+ * @brief Utility for storing command line arguments
+ */
+class Options {
+    char **begin;
+    char **end;
+
+    /**
+     * @brief Get the char* value of the cmd line argument
+     * @param  option           the argument in cmd
+     * @return char*
+     */
+    char *get_cmd_option(const std::string &option) {
+        char **itr = std::find(begin, end, option);
+        if (itr != end && ++itr != end) {
+            return *itr;
+        }
+        return 0;
+    }
+
+    /**
+     * @brief Get the int type value of cmd line argument
+     * @param  option           the cmd line argument
+     * @return int              the int type value of cmd line argument 'option'
+     */
+    int get_cmd_line_argument_int(const std::string &option) {
+        if (char *value = get_cmd_option(option)) {
+            return std::stoi(value);
+        }
+        return 0;
+    }
+
+    /**
+     * @brief Get the string type value of cmd line argument
+     * @param  option           the cmd line argument
+     * @return std::string      the int type value of cmd line argument 'option'
+     */
+    std::string get_cmd_line_argument_string(const std::string &option) {
+        if (char *value = get_cmd_option(option)) {
+            return std::string(value);
+        }
+        return "";
+    }
+
+  public:
+    int num_test;
+    int warm_up;
+    int num_in_step;
+    int random_seed;
+    std::string para_info_json;
+
+    /**
+     * @brief Construct a new Command Line object
+     * @param  argc
+     * @param  argv
+     */
+    Options(int argc, char *argv[]) {
+        begin = argv;
+        end = argv + argc;
+        num_test = get_cmd_line_argument_int("--num_test");
+        num_test = (num_test == 0 ? 1 : num_test);
+        warm_up = get_cmd_line_argument_int("--warm_up");
+        warm_up = (warm_up == 0 ? 1 : warm_up);
+        num_in_step = get_cmd_line_argument_int("--num_in_step");
+        num_in_step = (num_in_step == 0 ? 100 : num_in_step);
+        random_seed = get_cmd_line_argument_int("--random_seed");
+        random_seed = (random_seed == 0 ? time(NULL) : random_seed);
+        para_info_json = get_cmd_line_argument_string("--config_json");
+        para_info_json =
+            para_info_json == ""
+                ? R"({"algo":0,"arrayLength":2,"conv_type":0,"dilationA":[1,1],"filterStrideA":[1,1],"filter_dims":[32,128,3,3],"input_dims":[32,128,14,14],"input_stride":[25088,196,14,1],"input_type":0,"mode":1, "name":"cudnnConvolutionBackwardFilter","output_dims":[32,32,14,14],"output_stride":[6272,196,14,1],"padA":[1,1],"use_tensor_core":false})"
+                : para_info_json;
+    }
+};
+
+namespace cudnn_test {
```

Comment:
Why is Option not in the namespace cudnn_test?

Reply:
added option in cudnn_test namespace